### PR TITLE
dcos-2405 add --package-version option to `package describe`

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: cli/dcoscli/data/help/package.txt [hunk 1]
```diff
@@ -24,7 +25,7 @@ Options:
     --info                                 Show a short description of this subcommand
     --options=<file>                       Path to a JSON file containing package
                                            installation options
-    --package-version=<package_version>    Package version to install
+    --package-version=<package_version>    Only apply to specific package version
```
Comment (mgummelt) [inline cli/dcoscli/data/help/package.txt]: s/specific/this

File: cli/tests/integrations/test_package.py [hunk 2]
```diff
@@ -146,6 +146,14 @@ def test_describe():
...
 
 
+def test_describe_specific_version():
+    stdout = file_bytes(
+        'tests/data/package/json/test_describe_marathon_0.8.1.json')
+    assert_command(['dcos', 'package', 'describe', '--package-version=0.8.1',
+                    'marathon'],
+                   stdout=stdout)
+
+
```
Comment (mgummelt) [inline cli/tests/integrations/test_package.py]: test invalid version

File: cli/dcoscli/data/help/package.txt (hunk 1 above)
Comment (tamarrow) [inline cli/dcoscli/data/help/package.txt]: done

File: cli/tests/integrations/test_package.py (hunk 2 above)
Comment (tamarrow) [inline cli/tests/integrations/test_package.py]: done

**Comment by tamarrow** [comment]
@mgummelt updated

File: cli/dcoscli/data/help/package.txt [hunk 3]
```diff
@@ -24,7 +25,7 @@ Options:
     --info                                 Show a short description of this subcommand
     --options=<file>                       Path to a JSON file containing package
                                            installation options
-    --package-version=<package_version>    Package version to install
+    --package-version=<package_version>    Only apply to this package version
```
Comment (ssk2) [inline cli/dcoscli/data/help/package.txt]: I'd change `this` to `specified` for...more specificity.

File: cli/dcoscli/package/main.py [hunk 4]
```diff
@@ -174,14 +177,19 @@ def _describe(package_name, cli, app, options_path):
     if pkg is None:
         raise DCOSException("Package [{}] not found".format(package_name))
 
-    # TODO(CD): Make package version to describe configurable
-    pkg_revision = pkg.latest_package_revision()
+    pkg_revision = pkg.latest_package_revision(package_version)
+
+    if pkg_revision is None:
+        raise DCOSException("Package [{}] not available with version {}".format
```
Comment (ssk2) [inline cli/dcoscli/package/main.py]: Reword this error slightly. `Version {} of package [{}] is not available.`

File: cli/dcoscli/data/help/package.txt (hunk 3 above)
Comment (tamarrow) [inline cli/dcoscli/data/help/package.txt]: lol: https://github.com/mesosphere/dcos-cli/pull/300#discussion_r36364623

File: cli/tests/data/package/help.txt [hunk 5]
```diff
@@ -24,7 +25,7 @@ Options:
     --info                                 Show a short description of this subcommand
     --options=<file>                       Path to a JSON file containing package
                                            installation options
-    --package-version=<package_version>    Package version to install
+    --package-version=<package_version>    Only apply to this package version
```
Comment (ssk2) [inline cli/tests/data/package/help.txt]: Same as previous comment. `this` to `specified`.

File: cli/dcoscli/data/help/package.txt (hunk 3 above)
Comment (ssk2) [inline cli/dcoscli/data/help/package.txt]: Hmm, thoughts @jsancio? We have two votes for `speci*` and one for `this`.

Comment (mgummelt) [inline cli/dcoscli/data/help/package.txt]: no need for argument.  I relent.

Comment (mgummelt) [inline cli/dcoscli/data/help/package.txt]: but I'm right and you're wrong

File: cli/dcoscli/package/main.py (hunk 4 above)
Comment (tamarrow) [inline cli/dcoscli/package/main.py]: done

**Comment by tamarrow** [comment]
@ssk2 updated

**Comment by ssk2** [comment]
@tamarrow - looks good to me. Please merge when TeamCity is happy (unsure why it's unhappy).

**Pull request merged**